src/poetry/installation/executor.py

```diff
@@ -906,3 +898,12 @@ def _create_directory_url_reference(self, package: Package) -> dict[str, Any]:
             "url": Path(package.source_url).as_uri(),
             "dir_info": dir_info,
         }
+
+    def _get_archive_info(self, package: Package) -> dict[str, Any]:
+        archive_info = {}
```

Comment:
Can add a docstring with a link to https://packaging.python.org/en/latest/specifications/direct-url/ and info that it supersedes PEP 610, please?